lbaas/origin_server_pool_test.go

```diff
@@ -0,0 +1,188 @@
+package lbaas
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/go-oracle-terraform/helper"
+	"github.com/hashicorp/go-oracle-terraform/opc"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test the Origin Server Pool lifecycle to create, get, delete a Origin Server
+// Pool and validate the fields are set as expected.
+func TestAccOriginServerPoolLifeCycle(t *testing.T) {
+	helper.Test(t, helper.TestCase{})
+
```

Comment:
with testify I frequently see people reassign `assert` like: `assert := assert.New(t)` wrapping the `t` so you don't have to pass it continuously below.

So then you could just do: `assert.Equal(expected, actual)`